pkg/urbit/daemon/main.c

```diff
@@ -1196,10 +1196,32 @@ _cw_utils(c3_i argc, c3_c* argv[])
   return 0;
 }
 
+/* _main_bin_path(): copy binary path, ensuring ".exe" ending on windows
+*/
+static c3_c*
+_main_bin_path(c3_c* bin_c)
+{
+#ifdef U3_OS_mingw
+  c3_w len_w = strlen(bin_c);
+
+  if ( (4 <= len_w) && !strcmp(bin_c + len_w - 4, ".exe") ) {
```

Comment:
The `strcmp` return value check appears to be wrong, which seems to be causing the CI failure on MinGW. It should be `0 != strcmp(bin_c + len_w - 4), ".exe")`.